frontend/controller/router.js

```diff
@@ -178,4 +179,8 @@ router.beforeEach((to, from, next) => {
   next()
 })
 
+sbp('sbp/selectors/register', {
+  'state/router': () => router
```

Comment:
But this file is `controller/router.js`. I didn't understand where you want to place this. 🤔

I placed it here because it's the closest possible place to the `router` creation.

Reply:
Sorry, I wasn't referring to the file location, but to the SBP namespace. My suggestion here is to rename line 183 to `'controller/router'`. The file/folder path is also named this way because conceptually the router is part of the C in MVC. SBP namespaces are also based on conceptual hierarchies, so I think it would make sense to also have the SBP namespace here be `'controller/`'

Reply:
Ah! 🤦‍♀ done!